ui/desktop/src/goosed.ts

```diff
@@ -282,46 +269,31 @@ export const startGoosed = async (
   // Wait for the server to be ready
   const isReady = await checkServerStatus(port);
   log.info(`Goosed isReady ${isReady}`);
-  if (!isReady) {
-    log.error(`Goosed server failed to start on port ${port}`);
+
+  const try_kill_goose = () => {
     try {
       if (isWindows) {
-        // On Windows, use taskkill to forcefully terminate the process tree
-        // Security: Validate PID is numeric and use safe arguments
         const pid = goosedProcess.pid?.toString() || '0';
-        if (!/^\d+$/.test(pid)) {
```

Comment:
will this be needed on windows?

Reply:
ah - centralising it

Reply:
well, it strikes me that either we can kill the thing or we can't. why would we try to guess what a pid looks like if we got it from the os and pass it to the os?